spec/tbd_tests_spec.rb

```diff
@@ -2404,1851 +2404,6 @@
     end
   end
 
-  it "can process thermal bridging and derating: LoScrigno" do
```

Comment:
Served its (short-term) purpose. Moving the RSpec back to TBD Test [Suite](https://github.com/rd2/tbd_tests/pull/26).